sdk/build.gradle

```diff
@@ -10,8 +10,8 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 27
-        versionCode 38
-        versionName "0.5.9"
+        versionCode 39
+        versionName "0.5.10"
```

Comment:
Should these versions be in sync?
https://github.com/talkable/android-sdk/blob/5808fcdfa1456236dac8982bc24418e26b9dac4a/app/build.gradle#L21-L22

Reply:
No, this is needed only when `app` code is changed